src/java.desktop/share/classes/javax/swing/text/MaskFormatter.java

```diff
@@ -100,7 +100,6 @@
  * <pre>
  *   MaskFormatter formatter = new MaskFormatter("###-####");
  *   formatter.setPlaceholderCharacter('_');
- *   formatter.getDisplayValue(tf, "123");
```

Comment:
This code is part of an example, without this line, the following lines of documentation do not make sense. Probably, `getDisplayValue` should be replaced with `valueToString`. You should check that and ensure the sample and the expected output, as discussed in the following sentences, is correct.

Reply:
I agree. Simply deleting this line solves nothing. The example needs to be reviewed and the text needs to say something correct and useful.

Reply:
Yeah, I've checked with `valueToString` method, its working fine. I'll replace `getDisplayValue` with `valueToString`? ValueToString takes one parameter which is of Object type. It return the formated value.
Sample Code:
        MaskFormatter formatter = new MaskFormatter("###-##");
        formatter.setPlaceholderCharacter('*');
        System.out.println(formatter.valueToString("87"));

Output:
87*-**

Reply:
Yes, this will make the example correct.

This sample code demonstrates what is discussed in the docs:
```java
MaskFormatter formatter = new MaskFormatter("###-####");
formatter.setPlaceholderCharacter('_');
System.out.println(formatter.valueToString("123"));

formatter = new MaskFormatter("###-####");
formatter.setPlaceholderCharacter('*');
formatter.setPlaceholder("555-1212");
System.out.println(formatter.valueToString("123"));
```
and produces the output:
```
123-____
123-1212
```
which perfectly aligns with the documentation.

Reply:
I have updated the PR.

Reply:
@prrace The PR has been reviewed and approved by @prsadhuk and @aivanov-jdk...... Can I go ahead and integrate it?